jf_agent/main.py

```diff
@@ -238,6 +238,11 @@ def _check_and_get(envvar_name):
         }
     elif git_provider == 'gitlab':
         return {'gitlab_token': _check_and_get(f'{prefix}GITLAB_TOKEN')}
+    elif git_provider == 'gitlab_v3':
```

Comment:
Might as well just combine this into the same statement: `elif git_provider in ('gitlab', 'gitlab_v3'):`

Reply:
done (& tested)